lambda/index.ts

```
@@ -1,48 +1,105 @@
+    throw new Error('TOPIC_ARN environment variable is not set');
+  }
 
   await Promise.all(
-    emailsToSend.filter(Boolean).map((email) =>
+    emailsToSend.filter(Boolean).map((email: string | null) =>
       sns.send(
         new PublishCommand({
           TopicArn: topicArn,
           Subject: 'EyePatch - Alerta de Status',
-          Message: email ?? undefined,
+          Message: email!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)